integration/Dunet.Integration.csproj

```
@@ -9,7 +9,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Dunet" Version="1.10.0" />
+        <PackageReference Include="Dunet" Version="1.11.0" />
```

```suggestion
        <PackageReference Include="Dunet" Version="1.10.0" />
```
Let's not bump this right now or it won't build in GH actions.

---

I know you probably bumped this for testing purposes, so consider this a reminder to revert before we can merge :)

---

Thanks, I've reverted that :D